### PR TITLE
feat(suite): tor splash screen uplift

### LINKED ISSUE #5805: Tor splashscreen on Suite launch
[Notion](https://www.notion.so/satoshilabs/Tor-Preflight-loader-Splash-screen-fd922b2d142c4d719f356e941b1bff71)

### PR DESCRIPTION
**Pull request by dahaca**
## Description

Improved the look of the splash screen. Decided not to move the button since it's cleaner that way + the [original design](https://www.figma.com/file/pIibgxmxD0jrAa9UGJd9da/Neue-Desktop-1.5?node-id=9538%3A83606) had it like that

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/5805

## Screenshots:

image

image

### CODE REVIEW

File: packages/suite/src/views/settings/general/Tor.tsx [hunk 1]
```diff
@@ -13,20 +13,36 @@ import { TorStatus } from '@suite-types';
...
     const { toggleTor } = useActions({
         toggleTor: suiteActions.toggleTor,
     });
 
     const { anchorRef, shouldHighlight } = useAnchor(SettingsAnchor.Tor);
 
+    useEffect(() => {
+        if (hasTorError) {
+            setTimeout(() => setHasTorError(false), 1000);
+        }
```
Comment (tomasklim) [inline packages/suite/src/views/settings/general/Tor.tsx:27]: Missing clearTimeout

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx [hunk 2]
```diff
@@ -189,42 +188,23 @@ export const TorLoadingScreen = ({ callback }: TorLoadingScreenProps) => {
                                 isRed={torStatus === TorStatus.Error}
                                 value={torStatus === TorStatus.Error ? 100 : progress}
                             />
+
                             <ProgressMessage>
                                 {torStatus === TorStatus.Error ? (
-                                    <P>Failed</P>
+                                    <span>Failed</span>
```
Comment (matejkriz) [inline packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx]: Can we make it translatable?

**Comment by tomasklim** [comment]
It will probably break #5912

File: packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx (hunk 2 above)
Comment (matejkriz) [reply packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx]: I mean the whole TorLoadingScreen. I know it's not a part of this PR... But maybe we could just wrap it by `IntlProvider` a provide the language somehow? 🤔

Comment (dahaca) [reply packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx]: 600b02475b6bd2834b30af81f5acccc9724fc91c

File: packages/suite/src/views/settings/general/Tor.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/views/settings/general/Tor.tsx:27]: 600b02475b6bd2834b30af81f5acccc9724fc91c

**Review by tomasklim — APPROVED** [review]
👍

**Comment by jerechinsky** [comment]
What's the context of the `TR_FAILED` string? Could you provide a screenshot please?

**Comment by jerechinsky** [comment]
@dahaca how would you personally translate it to russian? :)

**Comment by dahaca** [comment]
> @dahaca how would you personally translate it to russian? :) 

Запуск Tor / Активация Tor (возможно Tor'а?...)

Ошибка при запуске / Проблема при активации / Ошибка активации Tor

🤔🤔🤔

**Pull request merged**